docs/tutorials/browser/repl.qmd

```diff
@@ -16,14 +16,12 @@ penguins[^1] dataset loaded into the DuckDB backend!
 from urllib.parse import urlencode
 
 lines = """
-%pip install numpy pandas tzdata
-import pyodide_js, pathlib, js
-wheel_url = "https://storage.googleapis.com/ibis-wasm-wheels/pyarrow-17.0.0-cp311-cp311-emscripten_3_1_46_wasm32.whl"
-await pyodide_js.loadPackage(wheel_url)
+%pip install numpy pandas tzdata duckdb
+import micropip, pathlib, js, unix_timezones
```

Comment:
For some reason *importing* `unix_timezones` is required, because pyarrow's own import of it raises an exception. It doesn't really make much sense, but this works around the issue.

I'll report upstream.

Reply:
Reported upstream here: https://github.com/pyodide/pyodide/issues/5344